rich/traceback.py

```diff
@@ -416,6 +418,8 @@ def safe_str(_object: Any) -> str:
                     line=exc_value.text or "",
                     msg=exc_value.msg,
                 )
+            if isinstance(exc_value, RichRenderable):
```

Comment:
My concern here is that if we offer arbitrary renderables, then the Traceback object may no longer be trivially sererializeable. Which was one of the goals for this class.

And since a renderable may be mutable, it may become out of date if it isn't immediately printed.